docs/guides/service-discovery.md

```diff
@@ -36,30 +36,28 @@ For an app named `myapp` with a `convox.yml` like this:
         internal: true
         ports:
           - 5000
-          - 5001
       web:
         port: 3000
 
 You would see a `convox services` output similar to this:
 
     $ convox services
     SERVICE  DOMAIN                               PORTS
-    auth     auth.convox-myapp.svc.cluster.local  5000 5001
+    auth     auth.convox-myapp.svc.cluster.local  5000
     web      web.myapp.0a1b2c3d4e5f.convox.cloud  443:3000
 
-The `web` Service could reach the `auth` Service using the following endpoints:
+The `web` Service could reach the `auth` Service using the following URL:
 
-* `auth.convox-myapp.svc.cluster.local:5000`
-* `auth.convox-myapp.svc.cluster.local:5001`
+* `http://auth.convox-myapp.svc.cluster.local:5000`
 
-> Note that the internal `auth` Service is no longer receiving automatic HTTPS termination. If you want this
-> connection to be encrypted you would need to terminate HTTPS inside your Service.
+> Note that the internal port of the `auth` Service is not receiving automatic SSL termination. 
+> If you want this connection to be encrypted you would need to handle SSL inside the Service.
 
 DNS search suffixes are automatically configured for internal hostnames on a Rack. The following URLs would
 also work for contacting the `auth` Service:
 
 * `http://auth:5000` for Services on the same app.
 * `http://auth.convox-myapp:5000` for other apps on the same Rack.
 
-> The `convox` portion of the internal hostnames in the examples above is the name of the Rack.
+> The `convox` portion of these internal hostnames is the name of the Rack.
```

Comment:
Is it worth specifying this like:
http://{SERVICE NAME}.{RACK NAME}-{APP NAME}.svc.cluster.local:{PORT}
to make it clear?

Reply:
Well, and `http://{SERVICE NAME}:{PORT}` as well 😄

Reply:
I think it's more clear to have the specific examples.